scripts/coverage.py

```diff
@@ -76,7 +76,9 @@ def check():
     new = load_coverage_data()
 
     decreased = False
+    print('we are checking')
```

Comment:
j'te propose d'enlever le code de debug maintenant !

Reply:
Bah, la fonction pas couverte soit disant c'était "onCancelMove" (celle sans code dedans).
Donc j'en ai fait une abstraite à override obligatoirement, c'qui n'm'as obligé qu'à l'ajouter pour DemoCardWrapper (j'avais oublié qu'y avais un quatrième wrapper) et à la tester elle et p-e aussi à ogwc ou lgwc.
Whatever happends, we must please the code !
And we cannot disappoint father !